test/usecases/run/GetAllRunsUseCase.test.js

```diff
@@ -347,4 +347,15 @@ module.exports = () => {
         expect(runs).to.have.lengthOf(46);
         expect(runs.every((run) => requiredQualities.includes(run.runQuality))).to.be.true;
     });
+
+    it('should successfully return an array, only containing runs found with lhc period filter', async () => {
+        getAllRunsDto.query = {
+            filter: {
+                lhcPeriod: 'lhc22_b',
```

Comment:
For retrieving multiple runs it would be better to use a multi-value filter and retrieve runs based on 2 lhcPeriods

Reply:
Moreover, I think above in `GetAllRunsUseCase.js` you used for filter `lhcPeriods` but in here I see you are using the singular version `lhcPeriod`